docs/network_firewall.rst

```diff
@@ -486,6 +563,128 @@ message "The changes have been applied successfully". Once you've set up the
 firewall, exit the Unsafe Browser, and continue with the "Keeping pfSense up
 to date" section below.
 
+3 NIC Example (SG-3100)
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The below guide assumes you are using a 3 NIC firewall such as the SG-3100.
+While the SG-3100 has an integrated switch, you may need to add a switch to
+the LAN interface if you use a different firewall.
+
+Set Up LAN
+''''''''''
+
+Although we set up the LAN interface during the Setup Wizard we need to make
+a few revisions. Navigate in the WebGUI to configure the LAN interface. Go to
+**Interfaces ▸ LAN**, and ensure the **Enable Interface** box is checked. Use
+these settings:
+
+-  IPv4 Configuration Type: Static IPv4
+-  IPv4 Address: ``10.20.2.1`` (Application Gateway IP)
+
+Make sure that the CIDR routing prefix is correct (``/24``). Leave everything else
+as the default. **Save** and **Apply Changes**.
+
+|3 NIC LAN Interface|
+
+Disable Anti-Lockout Rule
```

Comment:
nit: At line 668, we refer to the (old) anti-lockout rule. Can we either move this anti-lockout related section down there or move the warning here to avoid confusion?